src/test/e2e/14_create_sha_index_test.go

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package test provides e2e tests for Zarf.
+package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateIndexShaErrors(t *testing.T) {
```

Comment:
For tests like these I see it more commonly to make them [table tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) - in this case I think it would improve readability and allow us to see what cases we are missing, but it isn't a big deal either. :)

Reply:
Yeah we don't usually do them in e2e tests, but this seems like a reasonable case that improves clarity. I like it

Reply:
update: love this :)